transcendence-app/src/main.ts

```diff
@@ -1,8 +1,10 @@
 import { NestFactory } from '@nestjs/core';
+import { NestExpressApplication } from '@nestjs/platform-express';
```

Comment:
What does this do? Why do we need to use NestFactory.create<> here instead of the way it was?

Reply:
[NestJS](https://docs.nestjs.com/first-steps#platform) is an abstraction over other Node HTTP frameworks. By default, it uses Express.

On line 7, we use 
```
app.enable('trust proxy');
```

Which is an [Express](https://expressjs.com/en/guide/behind-proxies.html) feature, and to use it, we have to import this module and specify the NestExpressApplication interface.